srcpkgs/lazarus/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'lazarus'
+pkgname=lazarus
+version=1.8.4
+revision=1
+wrksrc=lazarus
+build_style=gnu-makefile
+makedepends="fpc qt5pas qt5-x11extras-devel"
+depends="fpc gdb fpc-source qt5pas"
+short_desc="Cross-platform IDE for FreePascal"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="GPL-2, LGPL-2.1, modified LGPL-2.1, MPL"
```

Comment:
use SPDX short-identifiers

Reply:
Fixed (dropped the MPL, the synedit source is dual-licensed (MPL/LGPL and there is neither a file with the MPL license text in the tarball, nor a mention of the MPL version...).